doc/whats_new/v0.22.rst

```diff
@@ -60,6 +60,14 @@ Changelog
 - |Fix| :func:`utils.check_array` now correctly converts pandas DataFrame with
   boolean columns to floats. :pr:`15797` by `Thomas Fan`_.
 
+- |Fix| :func:`utils.check_is_fitted` accepts back an explicit ``attributes``
+  argument to check for specific attributes as explicit markers of a fitted
+  estimator. When no explicit ``attributes`` are provided, only the attributes
+  ending with a single "_" are used as "fitted" markers. The ``all_or_any``
```

Comment:
this should be ``that ends with a underscore and does not start with double underscore.``
I'll push directly.